### DISCUSSION TITLE
Add EVENT_AFTER_LOGIN For Redirection Per User Group

### DISCUSSION THREAD

**Discussion by chattervast** [craftcms/cms#11170] [Ideas] [answered]
On several sites I've needed to have a redirect based on the user group or run some service. After testing all of the available events, I found that the previously removed EVENT_AFTER_LOGIN was the best candidate. Is there some reason why this is no longer in the system and what the replacement for it is?

**Comment by brandonkelly** [comment, marked as answer]
There is still an `EVENT_AFTER_LOGIN` event. It lives on `yii\web\User`, like it always has.
